packages/wrangler/src/pages/dev.tsx

```diff
@@ -228,6 +230,23 @@ export const Handler = async ({
 		if (!existsSync(scriptPath)) {
 			logger.log("No functions. Shimming...");
 			scriptPath = resolve(__dirname, "../templates/pages-shim.ts");
+		} else {
+			const runBuild = async () => {
+				try {
+					workerJsBuild({
```

Comment:
```suggestion
					await workerJsBuild({
```

Ah, great, thanks for adding this!